.gitignore

```
@@ -83,4 +83,4 @@ config/environments/*.local.yml
 .framgia-ci-reports/
 .framgia-ci-result.temp.yml
 
-.vscode
+/.vscode
```

```suggestion
.vscode
```
Cái này giữ nguyên như cũ vẫn được mà đúng không?